src/DiffSharp.Core/Tensor.fs

```diff
@@ -584,6 +584,9 @@ type Tensor =
     /// <summary>Computes element-wise (\a &gt;= b\), returning a boolean tensor containing a <c>true</c> at each location where the comparison is true</summary>
     member a.ge(b:Tensor) = TensorC(a.primalRaw.GeTT(b.primalRaw))
 
+    /// <summary>Computes element-wise (\a &gt;= b\), returning a boolean tensor containing a <c>true</c> at each location where the comparison is true</summary>

```

Comment:
Typo `&gt;=` should be `=`

Reply:
@VisualMelon thank you for catching this!